### DISCUSSION TITLE
RNG Manipulation unpatch

### DISCUSSION THREAD

**Discussion by Jerasaurus** [PurpurMC/Purpur#1051] [__Completed__] [answered]
This [patch](https://github.com/PaperMC/Paper/blob/c034be8a825df3936bca531cb4b29d90a56b0d7e/Spigot-Server-Patches/0070-Use-a-Shared-Random-for-Entities.patch) patches RNG manipulation. Purpur server really appeals to me because its better than paper for technical mc and this is the last big feature a lot of technical mc players want.

**Comment by encode42** [comment, marked as answer]
Added a toggle in 44b914cfe15ddb13fa5b0080c29ae9353d3ce6d2. Though, this is untested, as I am not a technical player and do not know how to manipulate RNG.

**Reply by Jerasaurus** [reply]
Just tested - it works. You're a legend!

**Reply by symmetricalbyte** [reply]
gby (god bless you)
